core/src/main/scala/org/apache/spark/status/AppStatusStore.scala

```diff
@@ -112,10 +112,14 @@ private[spark] class AppStatusStore(
     }
   }
 
-  def stageAttempt(stageId: Int, stageAttemptId: Int, details: Boolean = false): v1.StageData = {
+  def stageAttempt(stageId: Int, stageAttemptId: Int,
```

Comment:
Changing the return type to (StageData, jobIds) might be simpler.

Reply:
Done